content/recipes/repl.md

```diff
@@ -0,0 +1,59 @@
+### Read-Eval-Print-Loop (REPL)
+
+REPL is a simple interactive environment that takes single user inputs, executes them, and returns the result to the user.
+REPL feature lets you inspect your dependency graph and call methods on your providers (and controllers) directly from your terminal.
+
+#### Usage
+
+The REPL functionality is enabled when you call `repl` over some root NestJS module like so:
+
+```typescript
+@@filename(repl)
+import { repl } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  await repl(AppModule);
+}
+bootstrap();
+@@switch
+import { repl } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  await repl(AppModule);
+}
+bootstrap();
+```
+
+And then start the app just like the usual:
+
+```bash
+$ npm run start --entryFile repl
+```
+
+> info **Hint** `repl` returns a [Node.js REPL server](https://nodejs.org/api/repl.html), thus you can customize it as you wish.
+
+#### Native functions
+
+The built-in NestJS REPL comes with few native functions that are globally available when you start the REPL.
+You can call `help()` to list them out.
+
+If you don't recall what's the signature (ie: expected parameters and return types) of some function, you can call `<function_name>.help`.
+For instance:
+
+```text
+> select.help
```

Comment:
Let's use a different function as an example (`select` will be very rarely used)